build/files.js

```diff
@@ -98,10 +98,47 @@ const headRegexp = /(^module.exports = \w+;?)/m
         + 'if (!EE.listenerCount) EE.listenerCount = function(emitter, type) {\n'
         + '  return emitter.listeners(type).length;\n'
         + '};\n/*</replacement>*/\n'
+      ]
+
+    , constReplacement = [
+        /const/g
+      , 'var'
+      ]
+
+    , bufferIsEncodingReplacement = [
+      /Buffer.isEncoding\((\w+)\)/
+    ,   '([\'hex\', \'utf8\', \'utf-8\', \'ascii\', \'binary\', \'base64\',\n'
+      + '\'ucs2\', \'ucs-2\',\'utf16le\', \'utf-16le\', \'raw\']\n'
+      + '.indexOf(($1 + \'\').toLowerCase()) > -1)'
+    ]
+
+    , requireStreamReplacement = [
+      /var Stream = require\('stream'\);/
+    ,  '\n\n/*<replacement>*/\n'
+      + 'var Stream;\n (function (){try{\n'
+      + 'Stream = require(\'st\' + \'ream\');\n'
+      + '}catch(_){Stream = require(\'events\').EventEmitter;}}())'
+      + '\n/*</replacement>*/\n'
+    ]
+
+    , isBufferReplacement = [
+      /(\w+) instanceof Buffer/
```

Comment:
buffer instanceof Buffer doesn't work in the browser, so replaced with Buffer.isBuffer